core/indexer/elasticsearchDatabase.go

```diff
@@ -277,18 +277,10 @@ func (esd *elasticSearchDatabase) SaveMiniblocks(header data.HeaderHandler, body
 		return
 	}
 
-	buffInsert, buffUpdate := serializeBulkMiniBlocks(header.GetShardID(), miniblocks)
-	if buffInsert.Len() != 0 {
-		err := esd.dbWriter.DoBulkRequest(&buffInsert, miniblocksIndex)
-		if err != nil {
-			log.Warn("indexing bulk of miniblock insert ", "error", err.Error())
-		}
-	}
-	if buffUpdate.Len() != 0 {
-		err := esd.dbWriter.DoBulkRequest(&buffUpdate, miniblocksIndex)
-		if err != nil {
-			log.Warn("indexing bulk of miniblock update ", "error", err.Error())
-		}
+	buff := serializeBulkMiniBlocks(header.GetShardID(), miniblocks)
+	err := esd.dbWriter.DoBulkRequest(&buff, miniblocksIndex)
```

Comment:
The condition should stay as a precaution, do not call es server if `len(buff)` is 0

Reply:
Not need anymore because few lines above is verified if len(miniblocks) == 0 and return